### ISSUE TITLE
Allow multiple API to either get called one by one or in parallel

### ISSUE THREAD

**Issue by ChakshuGautam** [flipkart-incubator/Poseidon#210]
I have a requirement when I want to get remote data from multiple endpoints and collate them. Is it possible to do it through poseidon. Also, it it possible to chain requests one after another or do more than 1 requests in parallel and collate the data?

**Comment by pmohankumar** [comment]
Yes this is possible in Poseidon via a declarative way in your API file. Please refer to sample app and doc around it [here](https://github.com/flipkart-incubator/Poseidon/tree/master/sample#what-it-does).

This feature of dispatch and compose is provided by the [Hydra](https://github.com/flipkart-incubator/hydra) library used in Poseidon.
